tests/test_project_service.py

```
@@ -0,0 +1,81 @@
+        json.dump(project_data, f)
+    # set env variable to temp file path
+    monkeypatch.setenv("PROJECT_FILE", str(file_path))
+    service = ProjectService()
+    project = service.load_project()
+    assert isinstance(project, Project)
+    assert project.business_summary == "Test Business Summary"
+
+
+
+# test get_project returns loaded project
+def test_get_project(monkeypatch):
```

Ruff (0.15.0)

[warning] Unused function argument: `monkeypatch`

(ARG001)